jib-gradle-plugin/src/test/resources/gradle/projects/simple/build.gradle

```diff
@@ -21,4 +21,7 @@ jib {
   extraDirectory {
     path = file('src/main/custom-extra-dir')
   }
+  container {
+    extraClasspath = ['/d1','/d2']
```

Comment:
Just realized `extraClasspath = ['/d1:/d2']` will work fine in the end. I feel it will be good to disallow that to prevent spreading confusion in usage. Given that this is Java classpath, it is not unreasonable to ban the use of `:` or `;` although they are valid filename characters on many common filesystems. People just shouldn't use them particularly for Java classpath in an image. What do you think?

Reply:
I've been mulling this over…  The `jib.container.extraClasspath` property also benefits from this, so users can use the more-natural looking `-Djib.container.extraClasspath=/d1:/d2:/d3` rather than comma-separated values.  Considering that we don't actually perform any interpretation or evaluation of these values, I'm inclined to _laissez faire_.

Reply:
Hmm... one concern is that people may do `/d1;/d2;/d3` on Windows (which won't work) after finding a usage of `/d1:/d2:/d3` on the Internet.

Reply:
I'm leaning towards brian's solution here. Yes the user can use `:` or `;`, but hopefully users who go looking for this kind of advanced config already know what they are doing?

Reply:
Alright, users will eventually figure out the right usage if it fails to work. I also think the simplicity in implementation is valuable.